src/UI/Buyer/src/app/app.module.ts

```diff
@@ -433,8 +433,9 @@ export class AppModule {
         prefix: `${this.appConfig.appID}buyer`.toLowerCase(),
       },
     })
-    translate.setDefaultLang('en')
-    translate.use('en')
+    translate.addLangs(this.appConfig.supportedLanguages)
+    translate.setDefaultLang(this.appConfig.defaultLanguage)
```

Comment:
It probably also makes sense to persist the language choice in both the cookies and on the user object. We need it on cookies because user might not be logged in, if they are though prefer to retrieve from user object and then finally if neither then use the above logic to persist for the first time.

Reply:
This has been completed and has been commented on separately, so I will resolve this.